lib/index.js

```diff
@@ -183,8 +186,10 @@ class ElectronDownloader {
       if (exists && !this.force) {
         debug('zip exists', this.cachedZip)
         this.checkIfZipNeedsVerifying(cb)
-      } else {
+      } else if (this.opts.autoDownload) {
         this.ensureCacheDir(cb)
+      } else {
+        cb(new Error('File: "' + this.cachedZip + '" does not exist locally and autoDownload is false'))
```

Comment:
Can use interpolation syntax here

Reply:
Does node 4 support that?

Reply:
Yup! Electron Packager uses it everywhere.